app/frontend/src/components/bcgov/BCGovFooter.vue

```diff
@@ -26,14 +26,13 @@ export default {
 
 <style scoped>
 .gov-footer.v-footer {
-  bottom: 0;
+  flex-shrink: 0;
```

Comment:
This should be under border-top (alphabetical ordering)

Reply:
i'm trying this